aiogram/api/client/bot.py

```diff
@@ -167,6 +181,93 @@ async def close(self) -> None:
         """
         await self.session.close()
 
+    @staticmethod
+    async def __download_file_binary_io(
+        destination: BinaryIO, seek: bool, stream: AsyncGenerator[bytes, None]
+    ) -> BinaryIO:
+        async for chunk in stream:
+            destination.write(chunk)
+            destination.flush()
+        if seek is True:
+            destination.seek(0)
+        return destination
+
+    @staticmethod
+    async def __download_file(
+        destination: Union[str, pathlib.Path], stream: AsyncGenerator[bytes, None]
+    ) -> None:
+        async with aiofiles.open(destination, "wb") as f:
+            async for chunk in stream:
+                await f.write(chunk)
+
+    async def download_file(
+        self,
+        file_path: str,
+        destination: Optional[Union[BinaryIO, pathlib.Path, str]] = None,
+        timeout: int = 30,
+        chunk_size: int = 65536,
+        seek: bool = True,
+    ) -> Optional[BinaryIO]:
+        """
+        Download file by file_path to destination.
```

Comment:
nice explanation, can we have it in one place only? :)

Reply:
do you mean to add that file_path can be obtained from the getFile method?

Reply:
Sorry, I was writing about docstring. can we have it in `docs/*` only? not to edit many times.

Reply:
But other methods are described in both docstring and docs/*
I don't think that "syncing" documentation takes much time, and that this "syncing" will be performed frequently. And it is more convenient for the user not to open the documentation on the site every time.

Reply:
I understand, but sometimes we understand to do such things :D
aren't other methods generated?

Yep, this is not very necessary, I get it